### DISCUSSION TITLE
有简单的非mock prover的例子吗

### DISCUSSION THREAD

**Discussion by Demian101** [z2o-k7e/halo2-step-by-step#39] [Q&A]
@DK : 
- https://github.com/privacy-scaling-explorations/snark-verifier/blob/main/snark-verifier/examples/evm-verifier.rs

@Frank Jz Liu
- https://github.com/privacy-scaling-explorations/halo2/blob/main/halo2_proofs/examples/serialization.rs
- ^ A standard plonk implementation in a PLONKish proof stack lol

**Comment by dk** [comment]
EWRONGDK :)
